src/app/auth/onboarding/index.tsx

```diff
@@ -117,7 +119,7 @@ function OnboardingScreen() {
 
   return (
     <ScrollView style={styles.container}>
-      <Text style={[globalStyles.h1, styles.h1]}>
+      <Text style={globalStyles.h1}>
```

Comment:
Put everything from this Text component to right before the `bottomCotainer` in a View (this is for the space between in the flexbox)

Reply:
This will make it so the page works for any phone screen